packages/gql/src/generator/GraphQlApiClientGenerator.ts

```diff
@@ -28,7 +28,19 @@ export const apiCall = async ({ query, variables, fragments }) => {
     const response = await API.graphql({ query: gql\`\${query.replace('...fragments', fragmentStr)}\`, variables });
     return { success: true, data: Object.values(response.data)[0] };
   } catch (e) {
-    return e.errors?.[0] || { success: false, message: 'Unknown error' };
+    if (!e.errors) {
+      return { success: false, message: 'Unknown error' };
+    }
+
+    const errors = e.errors.map(({ extensions, message }) => ({
+      message,
+      inputs: extensions.inputs,
+    }));
+
+    if (errors.length === 1 && errors[0].inputs.length === 1 && !errors[0].inputs[0].field) {
+      return { success: false, message: errors[0].inputs[0].message };
+    }
+    return { success: false, message: errors[0].message, errors };
```

Comment:
`{ success: false, errors }` is sufficient, please remove `message` from response

Reply:
Current action wrapper on GY still uses the message property. 

dispatch({ type: `${actionName}${ActionStatus.FAILED}`, payload: ex.message });